### DISCUSSION TITLE
[archived] "Argo Server SSO with Argo CD Dex" docs discussion

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#12916] [General]
# en/latest/argo-server-sso-argocd/

https://argo-workflows.readthedocs.io/en/latest/argo-server-sso-argocd/

**Comment by Bfault** [comment]
I think it needs some update as `server.config.dex\.config` is not valid anymore but `configs.cm.dex\.config`

**Reply by agilgur5** [reply]
Yea, someone noticed this [on CD's Slack](https://cloud-native.slack.com/archives/C01TSERG0KZ/p1701252663218859) too. It's on my todo list to update, but PR welcome in the meantime! Can cc me for review
